pkg/skaffold/config/options.go

```diff
@@ -25,3 +25,13 @@ type SkaffoldOptions struct {
 	CustomTag    string
 	Namespace    string
 }
+
+// Labels returns a map of labels to be applied to all deployed
+// k8s objects during the duration of the run
+func (opts *SkaffoldOptions) Labels() map[string]string {
+	labels := map[string]string{}
+	if opts.CustomTag != "" {
+		labels["custom_tag"] = "true"
```

Comment:
I'd remove this

Add

- namespace
- profiles
- cleanup

Reply:
profiles is a little special because this will just contain the list of all profiles specified on the CLI, but not which one was actually used to create each k8s API object. was planning on opening an issue to add support for profile labeling outside this PR